src/api/routes/articleRouter.js

```diff
@@ -181,4 +182,7 @@ articleRouter.post('/:slug/comments/:id/report',
   validateInputs('reportComment', ['commentReason']),
   reportCommentController.reportComment);
 
+articleRouter.get('/most/popular',
```

Comment:
@MCFrank16 we have been speaking about proper route naming. How do you think this route can be improved?

Reply:
yeah lemme change this.... 

> sorry